src/main/kotlin/io/github/mojira/arisa/domain/IssueUpdateContextCache.kt

```diff
@@ -0,0 +1,5 @@
+package io.github.mojira.arisa.domain
+
+import io.github.mojira.arisa.infrastructure.Cache
+
+object IssueUpdateContextCache : Cache<IssueUpdateContext>()
```

Comment:
Why is this needed?

Reply:
Because I don't feel like I want to pass the cache instance to every mapper function, so I just create it as a singleton.

Reply:
I think its better to pass it, that way we can swap implementations in a single place instead of multiple